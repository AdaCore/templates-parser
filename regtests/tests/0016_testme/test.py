from test_support import *

run('testme', ["testme27.tmplt"])
run('print_tree', ["testme27.tmplt"])
