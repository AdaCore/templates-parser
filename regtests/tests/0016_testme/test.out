


include2.tmplt

1> a value

2> 3

3> 4

4> 3

5> 3

ok

[INFO] testme27.tmplt
 -> include2.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[TEXT] 
[INCLUDE] include2.tmplt
    $0 = include2.tmplt
    $1 = @_VAR1_@
    $2 = @_VAR3_@
    $3 = @_MAT_@
  [INFO] include2.tmplt
  [C_INFO]  1 FALSE
  [TEXT] 
  [TEXT] @_$0_@
  [TEXT] 
  [TEXT] 1> @_$1_@
  [TEXT] 
  [TEXT] 2> @_$2'Length_@
  [TEXT] 
  [TEXT] 3> @_$3'Line_@
  [TEXT] 
  [TEXT] 4> @_$3'Min_Column_@
  [TEXT] 
  [TEXT] 5> @_$3'Max_Column_@
  [TEXT] 
  [IF] (@_$2'Length_@ > 1)
    [TEXT] ok
  [ELSE]
    [TEXT] nok
  [END_IF]
